Editor/Resources/EditorWindow/Pages/HelpAndDocumentationPage.uxml

```diff
@@ -9,38 +9,51 @@
             <ui:Label name="HelpPageDescription" text="Learn how to integrate and deploy games for hosting on GameLift. The GameLift service manages game server deployment, operation, and scaling.&#10;You can deploy custom-built game servers or use GameLift Realtime Servers to configure lightweight game servers for your game."/>
         </ui:VisualElement>
         <ui:VisualElement class="divider"/>
-        <ui:VisualElement class="separator separator--large separator--horizontal">
-            <ui:VisualElement class="card card--small" >
+        <ui:VisualElement class="separator separator--horizontal">
+            <ui:VisualElement class="card card--small">
                 <ui:Image name="HelpPageReportIssueIcon" class="card__icon"/>
                 <ui:Label name="HelpPageReportIssueLink" class="card__text" text="Report Issue   \uf08e"/>
+                <ui:Image name="HelpPageReportIssueExternalLink" class="card__icon--small"/>
             </ui:VisualElement>
             <ui:VisualElement class="card card--small">
-                <ui:Image name="HelpPageDocumentationIcon" class="card__icon--small"/>
-                <ui:Label name="HelpPageDocumentationLink" class="card__text--small" text="Documentation \uf08e"/>
+                <ui:Image name="HelpPageDocumentationIcon" class="card__icon"/>
+                <ui:Label name="HelpPageDocumentationLink" class="card__text" text="Documentation \uf08e"/>
+                <ui:Image name="HelpPageDocumentationExternalLink" class="card__icon--small"/>
             </ui:VisualElement>
             <ui:VisualElement class="card card--small">
                 <ui:Image name="HelpPageVideoTutorialIcon" class="card__icon--small"/>
                 <ui:Label name="HelpPageVideoTutorialLink" class="card__text--small" text="Video Tutorials  \uf08e"/>
+                <ui:Image name="HelpPageVideoTutorialExternalLink" class="card__icon--small"/>
             </ui:VisualElement>
             <ui:VisualElement class="card card--small">
                 <ui:Image name="HelpPageForumIcon" class="card__icon--small"/>
                 <ui:Label name="HelpPageForumLink" class="card__text--small" text="AWS Forum     \uf08e"/>
```

Comment:
Is `card__text--small` defined anywhere?

Reply:
no - these classes should be `card__text`